anyway/current_user_functions.py

```diff
@@ -1,10 +1,11 @@
+import typing
 from flask_login import current_user
 
 from anyway.app_and_db import db
 from anyway.models import UserOAuth
 
 
-def get_current_user_email():
+def get_current_user_email() -> typing.Optional[str]:
```

Comment:
it's better to if you do:
`from typing import Optional`
so you'll be able to return `..->Optional[str]`